client/packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -86,8 +86,8 @@ export const DateUtils = {
     differenceInDays(Date.now(), dateInputHandler(date)),
   startOfDay,
   startOfYear,
-  formatRFC3339: (date: Date) =>
-    isValid(date) ? formatRFC3339(date) : undefined,
+  formatRFC3339: (date?: Date | null) =>
```

Comment:
No argument and an argument with a value of `undefined` *should* be considered the same thing (and as far as I can tell JS treats them the same in almost all cases). So this is just a stylistic thing and the code base generally follows the `?:` syntax over `undefined` doesn't it?

Reply:
it depends how you want users to use your method. When having as `date: Date | null | undefined` the user must provide an argument and forgetting it is an error. In this case not providing and argument doesn't make sense and we should treat it as an error. So not a stylisitic thing but a way to avoid errors :)